draft-ietf-httpbis-message-signatures.md

```diff
@@ -1900,7 +2046,7 @@ Without a signature, the internal service would need to trust that the incoming
 NOTE: '\' line wrapping per RFC 8792
 
 "@path": /foo
-"@query": Param=value&pet=Dog
+"@query": param=Value&Pet=dog
```

Comment:
should there not have been a question mark there?

Reply:
@bblfish Yes, that is fixed in #1908, please review the new examples there if you can, and we can merge them.